tools/src/controller-gen/go.mod

```diff
@@ -6,7 +6,7 @@ require sigs.k8s.io/controller-tools v0.9.2
 
 require (
```

Comment:
Why is a mod bump needed?

Reply:
It shouldn't be - I thought I had reverted this, sigh.